### DISCUSSION TITLE
How can I change the log files?

### DISCUSSION THREAD

**Discussion by alphabravocharly72** [jomjol/AI-on-the-edge-device#2684] [General]
Hello, Working good so far. But I have one measurement with 1000L and this is too much. How can I change or delete this value, so I have a proper value in HA? In the log file on the SD card? Or somewhere in HA?
![Screenshot 2023-11-07 222423](https://github.com/jomjol/AI-on-the-edge-device/assets/59312982/9ce7da56-cd84-44f4-a83a-2495ebddfe4f)

Thx
alpha

**Comment by caco3** [comment]
Your screenshots shows that the data is already in HA. So you would need to do it there.
**My suggestion: do not mess with the data in HA. I tried it many times and always broke it.**

But it does not look like a glitch, the data continues high after the step! I suggest to enable `MaxRateValue` to prevent accidental glitches, see https://jomjol.github.io/AI-on-the-edge-device-docs/Correction%20Algorithm/#maxratevalue-and-maxratetype

**Comment by alphabravocharly72** [comment]
how can I modify the data in HA? Just want delete this entry.

**Reply by caco3** [reply]
That's tricky, like said, I never succeeded.
Please ask in the HA community for help.

What I usually did was to rename the sensor and use a new entity :(

**Comment by alphabravocharly72** [comment]
ok, thx for your post
